roles/dnsmasq/molecule/default/converge.yml

```diff
@@ -100,18 +111,9 @@
              'artifacts') | path_join
           }}
         fname: "{{ item | basename }}"
```

Comment:
I think `| basename` acts weirdly in ansible when applied to a directory:
```
The result of this filter is different from the Unix basename program;
where basename for /foo/bar/ returns bar, the basename filter returns an empty string ('').
```

[source](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/basename_filter.html#notes)